### DISCUSSION TITLE
Enable to hide the sidebar of blog and release its space.

### DISCUSSION THREAD

**Discussion by Ouch1978** [facebook/docusaurus#5150] [Ideas]
For now, even I set blogSidebarCount: 0 to hide the content of the blog sidebar, the content width still not get increase.
I think the user experience will be better if we could get additional width for the content area when the sidebar is hidden just like docs do.

**Comment by lex111** [comment]
It is unlikely that we can place button to hide sidebar on blog pages, but make sense to increase blog content area a bit (suppose, from `col-8` to `col-10`) if `blogSidebarCount = 0`.
